.circleci/config.yml

```diff
@@ -564,6 +564,19 @@ jobs:
             # For more on twine, see:
             #   https://twine.readthedocs.io/en/latest/
             make schemas_deploy_pypi
+      - run:
+          name: Create the NPM distribution files
+          command: |
+            make schemas_build_npm
```

Comment:
So then we can take this out here since by the time it merges the file should be up to date?